src/avfoundation.cs

```diff
@@ -9145,13 +9167,27 @@ interface AVCapturePhotoSettings : NSCopying
 		[Export ("availableEmbeddedThumbnailPhotoCodecTypes")]
 		NSString[] _GetAvailableEmbeddedThumbnailPhotoCodecTypes { get; }
 
+		[Obsoleted (PlatformName.iOS, 12, 0, message: "Use AvailableEmbeddedThumbnailPhotoCodecTypes instead.")]
 		[iOS (11, 0)]
 		[Wrap ("Array.ConvertAll (_GetAvailableEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
 		AVVideoCodecType[] GetAvailableEmbeddedThumbnailPhotoCodecTypes { get; }
 
+		[iOS (11, 0)]
+		[Wrap ("Array.ConvertAll (_GetAvailableEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
+		AVVideoCodecType[] AvailableEmbeddedThumbnailPhotoCodecTypes { get; }
+
+#if XAMCORE_4_0
+		[iOS (11, 0)]
+		[NullAllowed, Export ("embeddedThumbnailPhotoFormat", ArgumentSemantic.Copy)]
+		NSDictionary NativeEmbeddedThumbnailPhotoFormat { get; set; }
+
+		[Warap ("NativeEmbeddedThumbnailPhotoFormat")]
```

Comment:
`Warap` won't compile

Reply:
Ah! Good catch, I compiled the project but did not see the error due to the XAMCORE if statement.